### ISSUE TITLE
View pending users invited by email to an app

### ISSUE THREAD

**Issue by bobeal** [ozwillo/ozwillo-portal#369]
As an improvement of #207.

Follow details provided by @tbroyer :

> It's a /apps/pending-acl/instance/{instance_id} and is modeled after the organization invitation
> See https://github.com/ozwillo/ozwillo-kernel/commit/a1e9c4021526c219f53074e5d4c676b983388890 for the details.

**Comment by bobeal** [comment]
Hi @tbroyer,

I'm facing a problem when trying to remove an user invited by email who has not yet accepted the invitation.

When calling the kernel to delete the invitation, I use the pending_entry_uri and pending_entry_etag data received in the user's ACE and got a 404 in return : 

```
| DEBUG| 19:08:52.795 | org.oasis_eu.spring.util.KernelLoggingInterceptor - Intercepted DELETE request to: https://kernel.ozwillo-dev.eu/apps/pending-acl/ace/80da926c-458d-44b9-9511-ad9a315fd9b6
| DEBUG| 19:08:52.796 | org.oasis_eu.spring.util.KernelLoggingInterceptor - Request body:
| DEBUG| 19:08:52.852 | kernelLogging.logRequestTimings - DELETE Request to https://kernel.ozwillo-dev.eu/apps/pending-acl/ace/80da926c-458d-44b9-9511-ad9a315fd9b6 took 56 ms
| DEBUG| 19:08:53.007 | org.oasis_eu.spring.util.KernelLoggingInterceptor - Response: 404 Not Found
```

**Comment by tbroyer** [comment]
OK, took me time to finally see the obvious; should be fixed by ozwillo/ozwillo-kernel@1697308361484767894d57d1a67a8a5c96f6be05

**Comment by bobeal** [comment]
It works. Thanks.
